src/main/resources/files/readme.md.tpl

```diff
@@ -107,4 +107,12 @@ Open [http://localhost:16686/](http://localhost:16686/) to see the traces. You h
 A type safe invocation of HTTP rest endpoints. Specification [here](https://microprofile.io/project/eclipse/microprofile-rest-client)
 
 The example calls one endpoint from another JAX-RS resource where generated Rest Client is injected as CDI bean.
+[/]
+
+[# th:if="${mp_graphql}"]
+### GraphQL
+
+GraphQL is a query language for APIs and a runtime for fulfilling those queries with your existing data. Specification [here](https://microprofile.io/project/eclipse/microprofile-graphql)
+
+The example contains a limited version of the SuperHero example from the TCK and shows the Schema file for it.
```

Comment:
I think we need to put more info to describe how the sample works. @andymc12 can you provide some texts to go into the readme so that anyone can follow the instruction and experiment with graphQL to put in queries and see the output?

Reply:
@Emily-Jiang I agree - would it make sense for me to open a separate issue and provide the text there (or in a separate PR)? There is a lot of good work in this PR, and I wouldn't want to hold it up on me writing a good example for the readme.

Reply:
sure, go ahead to open a new PR